packages/block-editor/src/store/defaults.native.js

```diff
@@ -15,12 +15,6 @@ const SETTINGS_DEFAULTS = {
 	__unstableGalleryWithImageBlocks: __DEV__,
 	alignWide: true,
 	supportsLayout: false,
-	__experimentalFeatures: {
-		color: {
-			text: true,
-			background: true,
-		},
-	},
```

Comment:
I most likely won't keep this because it probably disables text color settings for non-block themes, which is not wanted. See #34633 that introduced these lines I'm deleting. I just want to know if it fixes all the mobile unit and e2e tests, then I'll try to figure out a better fix.